.github/workflows/build_test.yml

```diff
@@ -253,7 +253,7 @@ jobs:
         docker run --name the_container --interactive -v /:/host arm32v7/ubuntu:focal /bin/bash -c "
           apt update &&
           apt install -y git python3 python3-dev python3-pip &&
-          pip3 install cython==0.29.30 setuptools\<49.2.0 hypothesis==6.23.3 pytest==6.2.5 &&
+          pip3 install cython==0.29.30 setuptools\<49.2.0 hypothesis==6.23.3 pytest==6.2.5 'typing_extensions>=4.2.0' &&
```

Comment:
If test_requirements.txt did not install mypy and pytest-cov for this platform (arm7), we could use it here, but maybe your way is easier. Maybe a comment before the docker command would help future reviewers?

Reply:
Like "keep in sync with 'test_requirements.txt'"

Reply:
Would it be worthwhile to something akin to a `test_minimal_requirements.txt` file? We can directly re-use this file inside `test_requirements.txt`, so It would allow us to get rid of this annoying dependency duplication.

Reply:
I personally find chaining dependencies annoying, so I would prefer a comment. If you feel strongly, go ahead, I will deal with it. This seems like the only place that would use `test_minimal_requirements.txt`.